frontend/src/pages/Screening.vue

```diff
@@ -1,40 +1,466 @@
+                      v-if="column.searchable"
+                      slot="searchable"
+                      slot-scope="props">
+                        <b-input
+                            v-model="props.filters[props.column.field]"
+                            icon="magnify"
+                            size="is-small" />
+                    </template>
+                    <template v-slot="props">
+                      <b-field>
+                          <b-select placeholder="Select Proficiency" v-model="props.row.EnglishProficiency" v-on:input="updateCourse">
+                            <option
+                              v-for="option in uniqueEnglishVals" :key="option" :value="option">
+                              {{option}}
+                            </option>
+                          </b-select>
+                      </b-field>
+                    </template>
+                    </b-table-column>
+                </template>
+
+
+
+              <!-- phone column --> 
+
+                <template v-for="(column, index) in columns" v-if="index == 5">
+                  <b-table-column :key="column.id" v-bind="column" sortable>
+                    <template
+                      v-if="column.searchable"
+                      slot="searchable"
+                      slot-scope="props">
+                        <b-input
+                            v-model="props.filters[props.column.field]"
+                            icon="magnify"
+                            size="is-small" />
+                    </template>
+                    <template v-slot="props">
+                        {{ props.row.PhoneNumber }}
+                    </template>
+                    </b-table-column>
+                </template>
+
+              <!-- blank ready to match column --> 
+
+                <template v-for="(column, index) in columns" v-if="index == 7">
+                  <b-table-column :key="column.id" v-bind="column" sortable>
+                    <template
+                      v-if="column.searchable"
+                      slot="searchable"
+                      slot-scope="props">
+                    </template>
+                    <template v-slot="props">                       
+                    </template>
+                    </b-table-column>
+                </template>
+            </b-table>
+
+        <!-- notification --> 
+        <b-notification v-model="isActive" aria-close-label="Close notification">
+            <b>Students ready to be matched?</b><br>
+            Confirming will add these students to the page for them to be matched with teachers.<br>
+            <button class="button field is-white">Cancel</button><button class="button field is-blue">Confirm</button>
+        </b-notification>
+
+      </section>
+    </div>
+  </Page>      
 </template>
 
+
 <script>
+
+// Placeholder for API //
+const API_URL = "http://localhost:3001/api/students/?status=Screening";
```

Comment:
Just start from `/api/...`

Reply:
Edited

Reply:
```suggestion
const API_URL = "/api/students/?status=Screening";
```
Is actually what I meant.